backend/tracim_backend/lib/webdav/authentification.py

```diff
@@ -65,7 +65,7 @@ def authDomainUser(
         api = UserApi(None, session, self.app_config)
         try:
             api.authenticate(
-                email=username,
+                email_or_username=username,
```

Comment:
personnally, i don't really like the "email_or_username" field. Naming this login and clarify thing seems to me better.

Reply:
changed int 49c8b1c5

Reply:
ok.